includes/support/abandoned-cart-hooks.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+function abandonedPluginHook($razorpayData) {
+
+     $rzpAbandonedData                                = array();
+     $rzpAbandonedData['woocommerceOrderID']          = $razorpayData['receipt'];
+     $rzpAbandonedData['amount']                      = $razorpayData['amount'];
+     $rzpAbandonedData['currency']                    = $razorpayData['currency'];
+     $rzpAbandonedData['customer_details']            = array();
+     $rzpAbandonedData['customer_details']['contact'] = $razorpayData['customer_details']['contact'];
+     $rzpAbandonedData['customer_details']['email']   = $razorpayData['customer_details']['email'];
```

Comment:
As email will be an optional field in the modal, will that affect the flow? varify this @angbpy

Reply:
This shouldn't affect the flow as email empty/not-empty checks are performed at the 3rd party plugin end before insertion

Reply:
What if the email key does not exist in rzp object? verify this flow.